magento/magento2
# magento/magento2#13295:Order grid page showing previous order id in U…

…RL while coming back from order view page

- fixed an issue with order id in URL when returning back to order grid page



### Description


### Fixed Issues (if relevant)

1. magento/magento2#13295:Order grid page showing previous order id in URL while coming back from order view page

### Manual testing scenarios

1. Open Order Page From order grid page
2. After order view page is opened click on the back button provided on action bar
3. When Order Grid page is opened then check the URL, it shouldn't have the order id in it.

### Contribution checklist
